RxCocoa/RxCocoa/iOS/UIBarButtonItem+Rx.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  UIBarButtonItem+RxExt.swift
+//  MVVM
+//
+//  Created by Daniel Tartaglia on 5/25/15.
+//
```

Comment:
Sry for asking this, but could we just make headers uniform with other headers. In this case it should probably be
```
//
//  UIBarButtonItem+Rx.swift <-- file name
//  RxCocoa <-- project name
//
//  Created by Daniel Tartaglia on 5/28/15. <-- this + git blame acknowledges and appreciates contribution to the project
//  Copyright (c) 2015 Krunoslav Zaher. All rights reserved. <-- I'm hoping this protects me from being sued by any contributor :), but will need to check to make sure that is the case
//
```
I'll probably change license later to be BSD, and compatible with other Rx distributions. MIT license this project is under shouldn't limit anyone from using it in any way.

Hope this is not a problem :) Maybe it looks pedantic, but really want to make this project great.

And I definitely need to investigate legal issues better.